spec/acceptance/golden_masters_spec.rb

```diff
@@ -1,29 +1,28 @@
 require 'spec_helper'
 
 describe 'Golden masters' do
-  let(:storage)         { RetirementSwap::Storage::Memory.new }
-  let(:output)          { RetirementSwap::Output::IOWriter.new(StringIO.new) }
-  let(:retirement_swap) { RetirementSwap::SwapAlgorithm.new(storage, output) }
-
-  it 'returns the correct results for the old spacewarps data' do
-    storage = RetirementSwap::Storage::Memory.new
-    output = RetirementSwap::Output::IOWriter.new(StringIO.new)
-    processor = RetirementSwap::Processor.new(storage, output, "52c1cf443ae7407d88000001" => {"algorithm" => "swap"})
-
-    classifications = File.readlines(File.expand_path("../../fixtures/spacewarps_ouroboros_classifications.json", __FILE__))
-                          .map {|line| JSON.parse(line) }
+  let(:storage) { RetirementSwap::Storage.new(DB) }
+  let(:output) { RetirementSwap::Output::IOWriter.new(StringIO.new) }
+  let(:processor) { RetirementSwap::Processor.new(storage, output, "52c1cf443ae7407d88000001" => {"algorithm" => "swap"}) }
 
+  after do
     verify do
-      classifications.map { |classification| processor.process(classification) }
-                     .map { |estimates| estimates.map { |estimate| [estimate.subject_id, estimate.user_id, estimate.answer, estimate.probability] } }
+      DB[:estimates].all.map do |row|
+        [[row[:subject_id], row[:user_id], row[:answer], row[:probability].round(20)]]
+      end
+    end
+  end
+
+  context 'with io and sequel' do
+    it 'works with the fully integrated kafka-sequel path' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.